docs/frameworks/laravel.md

```diff
@@ -18,104 +18,66 @@ composer require mnapoli/bref
-
-```dotenv
-VIEW_COMPILED_PATH=/tmp/storage/framework/views
-
-# We cannot store sessions to disk: if you don't need sessions (e.g. API)
-# then use `array`, else store sessions in database or cookies
-SESSION_DRIVER=array
-
-# Logging to stderr allows the logs to end up in Cloudwatch
-LOG_CHANNEL=stderr
-```
-
-Finally we need to edit `app/Providers/AppServiceProvider.php` because Laravel will not create that directory automatically:
-
-```php
-    public function boot()
-    {
-        // Make sure the directory for compiled views exist
-        if (! is_dir(config('view.compiled'))) {
-            mkdir(config('view.compiled'), 0755, true);
-        }
-    }
-```
-
-## Deployment
-
-At the moment deploying Laravel with its caches will break in AWS Lambda (because most file paths are different). This is why it is currently necessary to deploy without the config cache file. Simply run `php artisan config:clear` to make sure that file doesn't exist.
-
-Your application is now ready to be deployed. Follow [the deployment guide](/docs/deploy.md#deploying-with-sam).
+```
+ artisan vendor:publish --tag bref-sam-template
+```
+
+## Configuration
+You will need an S3 bucket to send the Function Package to in order for Cloudformation to consume it. Either use and existing bucket, or create a new one.
+```sh
+aws s3 mb s3://<bucket-name>
+```
+
+New edit your `.env` file and add:
+
+```ini
+BREF_NAME="<my-lambdas-name>"
+BREF_S3_BUCKET="<bucket-name>"
+```
+
+*OPTIONAL:* If you would like to do more advanced configuration you may publish the `bref.config` to your Laravel `./config` directory and edit it as well.
+```
+ artisan vendor:publish --tag bref-configuration
+```
+
+Lastly, tell Bref to finalize the SAM template for you.
+
+```
+artisan bref:config-sam
+```
+
+## Usage
+### Update Configuration
+This command can be run at anytime to update the template if you change environment variables, routes, or other configuration options.
+```
+artisan bref:config-sam
+```
+
+### Package Project
+This command will zip up your project and store it in the laravel `./storage` directory. It will also symlink `./storage/latest.zip` to the last zip package created.
+```
+artisan bref:package
+```
+### Deploy Project
+This command will deploy your project to SAM. The first time can take a moment. After this is run, your application should be up and running in AWS!
+```
+artisan bref:deploy
+```
+
+### Update Function Code
+Doing a full deploy to test code changes can be an aggravation. Use this command to update the Lambda Function in AWS after you make code changes. However, if you make configuration changes, you will need to run 'bref:deploy` again.
+```
+artisan bref:update
+```
+
+### Local API Testing
+If you have docker installed, and would like to test your code locally. This command will run your code in docker and give you a local testpoint to check it out in.
+```
+artisan bref:start-api
```

Comment:
just a thought: i wonder if this might be better as `artisan bref:start-app` instead. we know that under the hood it's calling `sam local start-api`, but as far as the end user is concerned it might be confusing to type `start-api` if they're not creating a rest api..?